x-pack/plugins/elastic_assistant/server/routes/post_actions_connector_execute.ts

```diff
@@ -58,6 +59,7 @@ export const postActionsConnectorExecuteRoute = (
           logger,
           request,
           elserId,
+          kbResource: ESQL_RESOURCE,
```

Comment:
Note, if we were to fetch `kbResource` via `getKbResource(request)` it would be undefined here as the other routes are using url params, not query params. Whomever needs to grab it from the client here will find that out though as well 🙂